modules/allometry/R/allom.BayesFit.R

```diff
@@ -126,7 +126,7 @@ allom.BayesFit <- function(allom, nrep = 10000, form = "power", dmin = 0.1, dmax
   }
   
   ## define priors
-  (s1    <- s2) <- 0.1  # IG prior on the within-study variance
+  s1     <- s2 <- 0.1  # IG prior on the within-study variance
```

Comment:
would it make sense to have this in the future be just two lines
s1 <- 0.1
s2 <- 0.1
just to make it easier to read.